modules/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-multiple-ingress-through-single-alb_{context}"]
+= Creating multiple ingresses through single ALB
+
+The user can route the traffic to multiple Ingresses through a single ALB.
+
+.Prerequisites
+
+.Procedure
+
+. Create an `AWSLoadBalancerController` and three `Ingress` resources:
++
+[source,yaml]
+----
```

Comment:
@xenolinux you will have to split up the yaml files because they are not rendered correctly.

Reply:
I removed the extra `---` and i think now it looks fine. But please let me know if that's not the case.